handlers.go

```diff
@@ -26,11 +26,13 @@ func (sf *Sunfish) AddFile(w http.ResponseWriter, r *http.Request) {
 	body, err := ioutil.ReadAll(io.LimitReader(r.Body, maxSiaFilesize))
 
 	if err != nil {
-		panic(err)
+		sf.logger.Println("ERROR: Could not ready body of request.")
```

Comment:
Is there a way that someone could manually trigger  this log message? Could I form a request that has no body, for example?